hq/app/config/Config.scala

```diff
@@ -110,8 +110,15 @@ object Config {
     } yield AwsAccount(id, name, roleArn, number)
   }
 
+  def getSnykConfig(config: Configuration): SnykConfig = {
+    SnykConfig(
+      SnykToken(requiredString(config, "snyk.token")),
+      SnykGroupId(requiredString(config, "snyk.organisation"))
```

Comment:
I'm not sure why this is called `GroupId` here and `organisation` in the private config; I've left that unchanged as I'm not too familiar with Snyk jargon